repo.go

```diff
@@ -82,3 +82,29 @@ func (c *Client) DeleteRepo(owner, repo string) error {
 	_, err := c.getResponse("DELETE", fmt.Sprintf("/repos/%s/%s", owner, repo), nil, nil)
 	return err
 }
+
+type MigrateRepoOption struct {
+	CloneAddr    string `json:"clone_addr" binding:"Required"`
```

Comment:
It's used higher up in the file as well, so I'm following convention here :-)

Want me to remove it?